hack/lib/etcd.sh

```diff
@@ -104,7 +109,7 @@ kube::etcd::start_scraping() {
   kube::log::info "Periodically scraping etcd to ${ETCD_SCRAPE_DIR} ."
   mkdir -p "${ETCD_SCRAPE_DIR}"
   (
-    while sleep 30; do
+    while sleep 10; do
```

Comment:
hmm - looking at this run:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/109202/pull-kubernetes-integration-go-canary/1509826346379907072

we were scraping at times:
```
1648807601.scrape
1648807614.scrape
1648807624.scrape
1648807694.scrape
```

so there is 70 seconds difference between 1648807624 and 1648807694

Does that mean that we are so cpu-starved that it may get delayed by a *minute*?

@aojea

Reply:
that maches some of my observations too, like the things inside the pod freeze ...